### ISSUE TITLE
Restore backup without the file directory

### ISSUE THREAD

**Issue by R7JANV1** [nextcloud/all-in-one#320]
Is this already possible?
If so, how?
And will it be included in the documentation?

**Comment by szaimen** [comment]
No, we will not add this option as the database is deeply linked to the existing files. 
Apart from that you can easily access the files in the backup archive at any time. and restore/view or do whatever you want with that.
See https://github.com/nextcloud/all-in-one#pro-tip-backup-archives-access
